digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/AssetDataImporterSpec.scala

```diff
@@ -400,7 +400,7 @@ class AssetDataImporterSpec extends FunSuite with Matchers {
     val linkId = 5170455
     val municipality = 853
     val obstaclePoint = Point(20, 20)
-    val obstacleType = 2
+    val pointAssetProperty = Property(1111, "suggest_box", "checkbox", false, Seq(PropertyValue("", None, false)))
```

Comment:
obscleType was deleted without a reason

check this could be the solution
```
val pointAssetProperty = Seq(Property(1111, "suggest_box", "checkbox", false, Seq(PropertyValue("", None, false))),
Property(2222, " obstacleType", "number", false, Seq(PropertyValue("2", None, false))))
```